client/actions/SignInAction.js

```diff
@@ -12,7 +12,9 @@ export default function userSigninRequest(userData) {
         data: res.data.user
       }, 'Success'));
       window.location = '/messageboard';

```

Comment:
i think this will force a full page reload, and i am not sure that's react way of navigating within pages, try to use `historyjs` or return the instance of axios, so in your component you can know when the request is successful then move to the desired page